core_dumps/core_dump_format.sh

```diff
@@ -29,3 +29,5 @@ echo "<b> Step: ${STEP_NAME}<br> Binary name: ${BINARY}<br> </b>" >> "${FILENAME
 save_ansi_to_html "Backtrace"
 save_ansi_to_html "Arguments"
 save_ansi_to_html "Locals"
+
+bzip2 --compress --best "${FILENAME}"
```

Comment:
Here we should compress ${COREDUMP}, not ${FILENAME}
COREDUMP is big blob, FILENAME here is lightweight HTML repore

Reply:
I am not sure actually!

https://github.com/mariadb-corporation/mariadb-columnstore-engine/blob/core-dumps-merge/core_dumps/core_dump_format.sh#L15

cat "$DUMPNAME" | bash "${SCRIPT_LOCATION}"/ansi2html.sh --palette=solarized >> "${FILENAME}"